src/ui/public/timepicker/__tests__/time_navigation.js

```diff
@@ -0,0 +1,38 @@
+import expect from 'expect.js';
+import moment from 'moment';
+import timeNavigation from '../time_navigation';
+
+describe('timeNavigation', () => {
+  let bounds;
+
+  beforeEach(() => {
+    bounds = {
+      min: moment('2016-01-01T00:00:00.000Z'),
+      max: moment('2016-01-01T00:15:00.000Z')
+    };
+  });
+
+  it('should step forward', () => {
+    const {from, to} = timeNavigation.stepForward(bounds);
+    expect(from).to.be('2016-01-01T00:15:00.000Z');
+    expect(to).to.be('2016-01-01T00:30:00.000Z');
+  });
```

Comment:
When I read this it took me a few seconds to figure out the meaning of this test, i.e. how much time is being stepped. Can we change the description to read: `should step forward by the amount of the duration`?